src/ifElse.test.js

```diff
@@ -1,11 +1,32 @@
+/* eslint-disable no-console */
 'use strict';
 
 describe('ifElse', () => {
-  // const { ifElse } = require('./ifElse');
+  const { ifElse } = require('./ifElse');
 
-  it('should ', () => {
+  it(`should call 'f' if first argument is 'true'`, () => {
+    const f = jest.fn();
+    const cb = () => console.log(2);
 
+    ifElse(() => true, f, cb);
+
+    expect(f).toHaveBeenCalled();
+  });
+
+  it(`should call 'f' if first argument is 'false'`, () => {
```

Comment:
The test description is incorrect. It should state that 'cb' is called if the first argument is 'false', not 'f'.